src/Uno.UWP/ApplicationModel/Contacts/ContactReader.Android.cs

```diff
@@ -0,0 +1,370 @@
+﻿
+#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Contacts
+{
+	public partial class ContactReader
+	{
+		internal ContactQueryOptions _queryOptions;
+		private Android.Database.ICursor _cursor = null;
+		private Android.Content.ContentResolver _contentResolver = null;
+
+		internal ContactReader(ContactQueryOptions options)
+		{
+			if (options is null)
+				throw new ArgumentNullException();
+
+			_queryOptions = options;
+
+			Android.Net.Uri oUri;
+			string sColumnIdName = "_id";

```

Comment:
Should it be a constant?

```suggestion
			string columnIdName = "_id";

```